src/codegen/mod.rs

```diff
@@ -1654,11 +1658,12 @@ impl CodeGenerator for CompInfo {
             fields.push(field);
         }
 
-        // Yeah, sorry about that.
-        let is_opaque = item.is_opaque(ctx, &());
         if is_opaque {
-            fields.clear();
-            methods.clear();
+            // Opaque item should not have generated methods, fields
+            debug_assert!(fields.is_empty());
+            debug_assert!(methods.is_empty());
+            // fields.clear();
+            // methods.clear();
```

Comment:
Nitpick: these commented out lines should be removed.

Reply:
Done!

Reply:
Done!